README.md

```diff
@@ -1,24 +1,97 @@
 
-Every pull request needs to be reviewed by another contributor to help with the overall quality of the documentation.
+## We are Astro
 
-## Running this project
+Astro is a site builder for the web platform.
+We want everyone to be successful building sites, and that means helping everyone understand how Astro works.
 
-- Clone the Project
+## You are Awesome
 
-  `git clone git@github.com:withastro/docs.git`
-- Run `yarn install` to install latest dependencies.
-  > This project uses yarn to manage dependencies. [Make sure that you have yarn v1 installed.](https://classic.yarnpkg.com/)
-- Run `yarn workspace docs dev` to start the dev server.
-- Run `yarn workspace docs build` to build the final site for production.
-  > The environment variable `SNOWPACK_PUBLIC_GITHUB_TOKEN` must be set to a personal access token with `public_repo` permissions to prevent rate-limiting.
+You can also help make the docs awesome.
+Your feedback is welcome.
+Your writing, editing, translating, designing, and developing skills are welcome.
+You being a part of our community is welcome.
 
-## Deploying
+## Chat with Us
 
-The site is automatically deployed when commits land in `latest`, via Netlify.
+You can learn more about Astro, get support, and meet other devs in [our Discord community](https://astro.build/chat).
 
-The "next" docs are automatically deployed when commits land in `main`, via Netlify at <https://main--astro-docs-2.netlify.app/getting-started/>.
+## Raise an Issue
+
+Is something missing?
+Is something wrong?
+Could something be better?
+Issues are a quick way for you to offer us feedback about the docs.
+
+Before you share, please [see if your issue has already been reported](https://github.com/withastro/docs/issues).
+
+## Edit a Page
+
+Every page on [docs.astro.build](https://docs.astro.build/) has an **Edit this page** button in the sidebar.
+You can click that button to edit the source code for that page in **GitHub**.
+
+After you make your changes, click **Commit changes**.
+This will automatically create a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) of the docs in your GitHub account with the changes.
+
+Once your edits are ready in GitHub, follow the prompts to **create a pull request** and submit your changes for review.
+Every pull request needs to be reviewed by our contributors.
+
+## Develop
+
+To begin developing locally, checkout this project from your machine.
+
+```shell
+git clone git@github.com:withastro/docs.git
+```
+
+You can install and run the project locally using your preferred package manager.
+
+```shell
+npm install
+
+npm start
+```
+
+If you’re copying these instructions, remember to [configure this project as a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork).
+
+```shell
+git remote add upstream git@github.com:csstools/docs.git
```

Comment:
really nice touch this bit

Reply:
I think this is absolutely a great goal, and very useful to have. I also think, eventually and if we find it comes up, it would be good to include specifically things like

- how we (decide whether to) merge PRs
- whether we routinely accept but might (often? to set up the expectation?) edit for style/consistency
- how likely a PR is to be accepted as-is, and what a "rejection" or "send-back" means and what they can do about it

 ... but we would need to discuss/decide those things first.  :)

We're at the stage where we're needing to get a consistent V1 version of the docs, and I worry that not every PR at this stage of the game is going to be immediately helpful as-is (through no fault of their own!) and runs the risk of removing consistency, rather than working towards it.

But personally, I DO want us to set the stage for as many (helpful!) community contributions as possible, where their pull requests are merged and they get the credit! Absolutely a style guide/contributors' handbook/checklist should be on the radar as much as the docs themselves are. I'm just hoping it doesn't come up too much until we're ready for it.  ;)

Reply:
Likewise, I feel the same, but I do believe that if the right organisational structures are in place prior to the tsunami of PR's or even a steady trickle appears would bode well for the overall success of the documentation platform. 
Im a big organisational guy, thinking towards this end the organisational structure of peer-review similar to the science and academic disciplines whereby the author of a PR can submit and is then sent back and forth until the document is correct and concise to fit with the body of existing documents would do well in this case,